templates/admin/config.tmpl

```diff
@@ -333,16 +333,6 @@
 
 				<div class="ui divider"></div>
 
-				<dt>{{.locale.Tr "admin.config.git_enable_reflogs"}}</dt>
-				<dd>{{if .Git.Reflog.Enabled}}{{svg "octicon-check"}}{{else}}{{svg "octicon-x"}}{{end}}</dd>
-
-				{{if .Git.Reflog.Enabled}}
-					<dt>{{.locale.Tr "admin.config.git_reflog_expiry_time"}}</dt>
-					<dd>{{.locale.Tr "tool.days" .Git.Reflog.Expiration}}</dd>
-				{{end}}
-
-				<div class="ui divider"></div>
-
```

Comment:
The git config parts are still shown, right? Care to show a screenshot?

Reply:
<img width="519" alt="image" src="https://github.com/go-gitea/gitea/assets/2114189/34505523-e5fa-4b30-83d5-bfe9acecc766">